server/db/seeders/20200131045539-demo-event.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('Events', [{
+      hostId: 1,
+      title: 'Support your brain health Event',
+      description: 'A strong memory depends on the health and vitality of your brain.' +
+        ' Whether you’re a student studying for final exams,' +
+        ' a working professional interested in doing all you can to stay mentally sharp,' +
+        ' or a senior looking to preserve and enhance your grey matter as you age,' +
+        ' there’s lots you can do to improve your memory and mental performance',
+      price: 30,
+      type: 0,
+      thumbnailUrl: 'https://content.fortune.com/wp-content/uploads/2020/01/Book-Review-Splendid-and-Vile-Uncanny-Valley-Ladys-Handbook.jpg',
```

Comment:
could we download some of these test images to a local server so we don't have to rely on external website?